experiment/run_experiment.py

```diff
@@ -481,27 +482,45 @@ def main():
                         '--experiment-name',
                         help='Experiment name.',
                         required=True)
-    parser.add_argument('-f',
-                        '--fuzzers',
-                        help='Fuzzers to use.',
-                        nargs='+',
-                        required=False,
-                        default=[])
-    parser.add_argument('-fc',
-                        '--fuzzer-configs',
-                        help='Fuzzer configurations to use.',
-                        nargs='+',
-                        required=False,
-                        default=[])
+    fuzzers_group = parser.add_mutually_exclusive_group()
+    fuzzers_group.add_argument('-f',
+                               '--fuzzers',
+                               help='Fuzzers to use.',
+                               nargs='+',
+                               required=False,
+                               default=None)
+    fuzzers_group.add_argument('-fc',
+                               '--fuzzer-configs',
+                               help='Fuzzer configurations to use.',
+                               nargs='+',
+                               required=False,
+                               default=[])
+    fuzzers_group.add_argument('-cf',
+                               '--changed-fuzzers',
+                               help=('Use fuzzers that have changed since the '
+                                     'last experiment. The last experiment is '
+                                     'determined by the database your '
+                                     'experiment uses, not necessarily the '
+                                     'fuzzbench service'),
+                               action='store_true',
+                               required=False)
+
     args = parser.parse_args()
 
-    if not args.fuzzer_configs:
-        fuzzer_configs = fuzzer_utils.get_fuzzer_configs(fuzzers=args.fuzzers)
-    else:
+    if args.fuzzer_configs:
```

Comment:
Should we add a error branch before this
if args.fuzzer_configs and args.changed_fuzzers:
  raise.....

Reply:
I think `parser.add_mutually_exclusive_group()` covers this.